docs/csharp/tutorials/exploration/patterns-objects.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Use patterns in objects - C# tutorial
+description: This tutorial teaches you techniques to use pattern matching in class members to create better models for object behavior
+ms.date: 11/05/2020
+---
+# Use pattern matching to build your class behavior for better code
+
+The pattern matching features in C# provide syntax to express your algorithms. You can use these techniques to implement the behavior in your classes. You can combine object-oriented class design with a data-oriented implementation to provide concise code while modeling real-world objects.
+
+In this tutorial, you'll learn how to:
+
+> [!div class="checklist"]
+>
+> - Express your object oriented classes using data patterns.
+> - Implement those patterns using C#'s pattern matching features.
+> - Leverage compiler diagnostics to validate your implementation.
+
+## Prerequisites
+
+You’ll need to set up your machine to run .NET 5, including the C# 9.0 compiler. The C# 9.0 compiler is available starting with [Visual Studio 2019 version 16.8](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019) or the [.NET Core 3.0 SDK](https://dotnet.microsoft.com/download).
```

Comment:
16.8 is currently preview. The link is for downloading the current non-Preview release (currently 16.7).